opswx-ckan-cookbook.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file doesn't need to be in version control.

Reply:
removed